### PR TITLE
Added JSON Config Files

### PR DESCRIPTION
**Pull request by AllenWang314**
Finished creating json config files for DDL, DML, DQL.

A set of ~60% of keywords used for DDL, DML, DQL will suffice to generate simple queries first. Later revisions can include adding more keywords and types of queries by modifying the json config files
A smaller set was used for the test json configs
Keywords and keyword mappings use old file paths and will be updated in a future PR, TODOs have been added to KeywordsMapping.java

### CODE REVIEW

File: tools/template_based_query_generation/src/main/java/KeywordsMapping.java [hunk 1]
```diff
@@ -1,122 +1,76 @@
+import com.google.common.collect.ImmutableList;
```
Comment (Bei95) [inline tools/template_based_query_generation/src/main/java/KeywordsMapping.java:1]: Please add some unit test for those newly added classes, or leave some note to add those tests in a future PR

File: tools/template_based_query_generation/src/main/java/Feature.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import java.util.List;
+
+/**
+ * Helper class that contains all keyword variants of a feature
+ */
+public class Feature {
+  private String feature;
```
Comment (bonancao) [inline tools/template_based_query_generation/src/main/java/Feature.java:7]: Please add some comments about those member variables.

File: tools/template_based_query_generation/src/main/java/Mapping.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
+import com.sun.tools.javac.parser.Tokens;
+
+import java.util.List;
+
+/**
+ * Helper class that lists PostgreSQL and BigQuery mappings and necessary tokens for all keyword variants
+ */
+public class Mapping {
+  private String postgre;
```
Comment (bonancao) [inline tools/template_based_query_generation/src/main/java/Mapping.java:9]: ditto

File: tools/template_based_query_generation/src/main/java/KeywordsMapping.java (hunk 1 above)
Comment (spoiledhua) [reply tools/template_based_query_generation/src/main/java/KeywordsMapping.java:1]: Done! Added TODOs for necessary unit tests.

**Comment by AllenWang314** [comment]
Files changed include .java and .json files. The java files are a working progress and TODOs have been left. The json files for the config follow the agreed format.

**Pull request merged**